app/helpers/ui_constants.rb

```diff
@@ -516,11 +516,11 @@ module UiConstants
     "vm-tags"                => PostponedTranslation.new(N_("Tagged %{tables}")) do
                                  {:tables => ui_lookup(:tables => "vm")}
                                 end,
-    "container_image-tags"   => PostponedTranslation.new(N_("Tagged %{tables}")) do
-                                 {:tables => ui_lookup(:tables => "container_image")}
+    "container_image-tags"   => PostponedTranslation.new(N_("Tagged %{models}")) do
+                                 {:models => ui_lookup(:models => "container_image")}
```

Comment:
There's no such model called `container_image`, nor is it present in `locale/en.yml` under `model`. What you
probably meant to use here was `ui_lookup(:models => "ContainerImage")`.